src/angular/chips/chips.scss

```diff
@@ -45,8 +45,8 @@
 
   height: var(--sbb-chips-chip-height);
   border-radius: var(--sbb-chips-chip-height);
-  color: var(--sbb-color-weak-accent);
-  background-color: var(--sbb-color-cloud);
+  color: var(--sbb-chips-color);
```

Comment:
You can leave `color: var(--sbb-color-weak-accent);` and remove `--sbb-chips-color` everywhere.
`--sbb-color-weak-accent` already has the correct colors in standard, lean and dark mode.